app/models/employer_events/employer_importer.rb

```diff
@@ -42,6 +42,26 @@ def plan_year_values
       end
     end
 
+    def issuer_ids 
+      @xml.xpath("//cv:benefit_groups", XML_NS).map do |node|
+        node.xpath("//cv:benefit_group/cv:elected_plans", XML_NS).map do |inner_node|
+          ids = inner_node.xpath("//cv:elected_plan/cv:carrier/cv:id/cv:id", XML_NS).map do |id|
+            stripped_node_value(id)
+          end
+         return ids.flatten || []
+        end
+      end
+    end
+
```

Comment:
Several issues here, and I'll add them under my notes for 40007 as well.

Remember that an XPATH prefix of "//" says "go all the way up to the root of the document, and match anything below anywhere you find it".  This means doing this will match benefit groups under OTHER plan years than the specific one you are trying to pull the IDs from - you are going to want to pass in the "plan_year" node you are looking for your issuer IDs under, then start your query against that node specifically with "**.**//" (note the dot, I tried to bold it, but it doesn't work super well in MD)

Reply:
created plan_year_loop method to scope to only the correct plan_year